packages/components/src/font-size-picker/index.js

```diff
@@ -68,11 +79,40 @@ export default function FontSizePicker( {
 
 	const fontSizePickerNumberId = `components-font-size-picker__number#${ instanceId }`;
 
+	const customInputProps = advancedFontSizing
+		? {
+				id: fontSizePickerNumberId,
+				className: classnames(
+					'components-font-size-picker__number',
+					advancedFontSizing &&
+						'components-font-size-picker__number--full-size'
+				),
+				onChange: ( event ) => onChange( event.target.value ),
+				ariaLabel: __( 'Custom' ),
+				value: value || '',
+				placeholder: 'e.g. calc(2rem - 13.5px)',
```

Comment:
Should be translatable

```suggestion
				placeholder: __( 'e.g. calc(2rem - 13.5px)' ),
```

Reply:
updated in `ade76846140e112e01ba2d1c09e8e9053940ac0b`